src/coreclr/jit/fginline.cpp

```diff
@@ -1325,6 +1325,15 @@ void Compiler::fgInsertInlineeBlocks(InlineInfo* pInlineInfo)
 
     lvaGenericsContextInUse |= InlineeCompiler->lvaGenericsContextInUse;
 
+    // If the inlinee compiler encounters switch tables, disable hot/cold splitting in the root compiler.
+    // TODO-CQ: Implement hot/cold splitting of methods with switch tables.
```

Comment:
Should we create tracking issues corresponding to the various TODO comments in the change?

Reply:
To add to this, the documentation for JIT support has some [future work items](https://github.com/dotnet/runtime/blob/main/docs/design/coreclr/jit/hot-cold-splitting.md#future-work) that may warrant their own issues.